### PR TITLE
ecg right sided leads

### PR DESCRIPTION
**Pull request by andyglittle**
references added

### CODE REVIEW

File: cards/ecg-right-posterior-leads/card.md [hunk 1]
```diff
@@ -52,3 +52,6 @@ Hints of an associated posterior infarct:
 2.  Upright T wave waves (in later stage)
 
 ## References
+
+1. [Edhouse J, Brady WJ, Morris F. ABC of clinical electrocardiography: Acute myocardial infarction-Part II. BMJ. 2002; 324: 963-6.] (https://doi.org/10.1136/bmj.324.7343.963)
```
Comment (jvoros) [inline cards/ecg-right-posterior-leads/card.md]: Like other review, this should be unordered list.

We are using a new routine in the build process that enforces some markdown formatting.

Looks like it requires a <tab> after the dash for lists, eg:

```
Here is a list

-    first item
-    second item
-    third item
```
Note tab spacing after the dash. Also note the blank line before the list.

Comment (dsifford) [inline cards/ecg-right-posterior-leads/card.md]: This requirement can be changed if you don't like it. A "tab" in this case is 3 spaces, not an actual tab. And technically speaking, the requirement is a tab-stop, not a full tab (details below).

The reason for it is to make sure that numbered list items stay aligned. 

Here's an example without it...

```
[ ... ]
8. List item 8
9. List item 9.
10. List item 10.
```

Here's an example with it...

```txt
[ ... ]
8.  List item 8.
9.  List item 9.
10. List item 10.
```

Comment (jvoros) [inline cards/ecg-right-posterior-leads/card.md]: Yeah, down in the dirt with remark now. I love the configurability for your own use case.

That convention of a tab stop is a little tricky to explain to non-coders. Lists in *Paucis Verbis* cards probably shouldn't be going longer than 10 items usually anyway. I think a single space is better.

Will it be hard to revert all the lists that got changed by your beautify process?

Comment (dsifford) [inline cards/ecg-right-posterior-leads/card.md]: @jvoros If you just remove the rule it will overlook it. No big deal at all.

Comment (jvoros) [inline cards/ecg-right-posterior-leads/card.md]: I actually just changed the rule. And then used an intermediate step with remark to rewrite the all the card files to comply to the new rule.

It was a piece of cake with Remark. Awesome package.

Comment (jvoros) [inline cards/ecg-right-posterior-leads/card.md]: I will change the Wiki with the new convention.

**Pull request merged**